File: MODUL-6/C/PRAK605-2310817220002-RaudatulSholehah.c
#include <stdio.h>
int main()
{
    int Matrix_1[10][10], Matrix_2[10][10], Hasil[10][10];
    int a, b, c, x, Total = 0;

    scanf("%d", &x);
    printf("Matriks A\n");

    for (a = 0; a < x; a++)
    {
        for (b = 0; b < x; b++)
        {
            scanf("%d", &Matrix_1[a][b]);
        }
    }
    printf("Matriks B\n");
    for (a = 0; a < x; a++)
    {
        for (b = 0; b < x; b++)
        {
            scanf("%d", &Matrix_2[a][b]);
        }
    }
    for (a = 0; a < x; a++)
    {
        for (b = 0; b < x; b++)
        {
            for (c = 0; c < x; c++)
            {
                Total = Total + Matrix_1[a][c] * Matrix_2[c][b];
            }
            Hasil[a][b] = Total;
            Total = 0;
        }
    }
    printf("Matriks AXB\n");
    for (a = 0; a < x; a++)
    {
        for (b = 0; b < x; b++)
        {
            printf("%d ", Hasil[a][b]);
        }
        printf("\n");
    }
    return 0;
}

File: MODUL-6/Python/PRAK605-2310817220002-RaudatulSholehah.py
x = int(input())
Matrix_1 = [];Matrix_2 = [];MatrixAXB = []

print('Matriks A')
for a in range(x):
    c =  list(map(int, input().split()))
    Matrix_1.append(c)

print('Matriks B')
for a in range(x):
    c =  list(map(int, input().split()))
    Matrix_2.append(c)

print('Matriks AXB')
for a in range(x):
    MatrixAXB.append([])
    for b in range(x):
        Total = 0
        for c in range(x):
            Total = Total + Matrix_1[a][c] * Matrix_2[c][b]
        MatrixAXB[a].append(Total)

for Isi in MatrixAXB:
    for c in Isi :
        print(c, end=" ")
    print()
